core/src/main/java/com/novoda/noplayer/mediaplayer/AndroidMediaPlayerImpl.java

```diff
@@ -264,11 +267,37 @@ public void selectAudioTrack(PlayerAudioTrack audioTrack) {
     }
 
     @Override
+    public boolean hasAvailableSubtitles() {
+        NoPlayerLog.w("Subtitles not implemented for Android Media Player");
```

Comment:
I think this should throw and the issue is actually in the `Player` factory and how it selects a `Player`. Perhaps the client should have a list of requirements of the `Player` they want and the `Factory` will select the most appropriate, similar to how it operates for `DRM`.